lib/src/commonMain/kotlin/app/moviebase/trakt/model/TraktMediaModel.kt

```diff
@@ -4,18 +4,18 @@ import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
-enum class TraktMediaType(val value: String) {
+enum class TraktMediaType(internal val type: String, internal val path: String) {
```

Comment:
Better to maintain both values separately, because they can have a different contexts. We have the `TraktListMediaType` enum.

Reply:
Done!